contrib/seeds/nodes_main.txt

```diff
@@ -1,188 +1,169 @@
+206.168.212.226:9999
+206.168.212.178:9999
+195.181.211.64:9999
+195.98.95.210:9999
+194.158.71.228:9999
+194.135.81.214:9999
+193.164.149.50:9999
+193.31.30.62:9999
+193.29.59.96:9999
+193.29.57.21:9999
+192.169.6.87:9999
+188.208.196.183:9999
+188.127.237.243:9999
+188.127.230.40:9999
+188.68.223.94:9999
+185.243.115.219:9999
+185.228.83.156:9999
+185.216.13.118:9999
+185.213.24.34:9999
+185.165.171.117:9999
+185.164.163.218:9999
+185.164.163.85:9999
+185.155.99.34:9999
+185.142.212.144:9999
+185.135.80.200:9999
+185.103.132.7:9999
+185.28.101.145:9999
+178.208.87.226:9999
+178.208.87.213:9999
+178.159.2.12:9999
+178.157.91.179:9999
+178.157.91.176:9999
+178.157.91.126:9999
+178.128.254.204:9999
+178.63.121.129:9999
+178.62.235.117:9999
+176.126.127.15:9999
+176.102.65.145:9999
+174.34.233.207:9999
+174.34.233.206:9999
+174.34.233.204:9999
+173.249.21.122:9999
+172.105.21.21:9999
+172.104.90.249:9999
+168.119.80.4:9999
+165.22.234.135:9999
+155.133.23.221:9999
+150.158.48.6:9999
+145.239.20.176:9999
+144.91.127.166:9999
+142.202.205.95:9999
+133.18.228.84:9999
+130.61.120.252:9999
+128.199.181.159:9999
+109.235.70.133:9999
+109.235.70.100:9999
+109.235.69.170:9999
+109.235.65.95:9999
+106.55.9.22:9999
+104.238.35.117:9999
+104.238.35.114:9999
+104.225.159.232:9999
+103.160.95.249:9999
+103.160.95.225:9999
+103.160.95.219:9999
+95.183.53.44:9999
+95.183.52.98:9999
+95.183.51.141:9999
+93.21.76.185:9999
+89.179.73.96:9999
+89.117.19.10:9999
+89.73.152.14:9999
+87.98.253.86:9999
+85.215.107.202:9999
+85.209.241.190:9999
+85.209.241.188:9999
+85.209.241.71:9999
+85.209.241.35:9999
+84.242.179.204:9999
+84.9.50.17:9999
+82.211.25.193:9999
+82.211.25.105:9999
+82.211.21.179:9999
+82.211.21.23:9999
+81.227.250.51:9999
+80.249.144.187:9999
+80.209.234.170:9999
+78.83.19.0:9999
+77.232.132.89:9999
+77.232.132.4:9999
+77.223.99.4:9999
+69.61.107.215:9999
+66.244.243.70:9999
+66.244.243.69:9999
+58.110.224.166:9999
+54.145.163.94:9999
+54.37.234.121:9999
+52.33.9.172:9999
+51.159.196.82:9999
+51.158.169.237:9999
+51.68.155.64:9999
+51.15.117.42:9999
+51.15.96.206:9999
+47.243.56.197:9999
+46.254.241.28:9999
+46.254.241.21:9999
+46.254.241.6:9999
+46.254.241.4:9999
+46.36.40.242:9999
+46.30.189.251:9999
+46.30.189.213:9999
+46.30.189.116:9999
+46.10.241.191:9999
+46.4.162.127:9999
+45.93.139.117:9999
+45.91.94.217:9999
+45.86.163.42:9999
+45.85.117.45:9999
+45.77.169.207:9999
+45.76.83.91:9999
+45.71.159.104:9999
+45.71.158.108:9999
+45.71.158.58:9999
+45.63.107.90:9999
+45.58.56.221:9999
+45.58.52.33:9999
+45.33.24.24:9999
+45.11.182.64:9999
+45.8.250.154:9999
+45.8.248.145:9999
+44.240.99.214:9999
+43.229.77.46:9999
+37.77.104.166:9999
+31.148.99.104:9999
+23.163.0.203:9999
+18.139.244.9:9999
+5.252.21.24:9999
+5.189.253.72:9999
+5.189.239.52:9999
+5.189.145.80:9999
+5.181.202.18:9999
+5.101.44.225:9999
+5.78.74.118:9999
+5.35.103.111:9999
+5.35.103.74:9999
+5.35.103.64:9999
+5.35.103.58:9999
+5.9.237.34:9999
+5.2.73.58:9999
+5.2.67.190:9999
```

Comment:
My `nodes_main.txt` is slightly different. Probably because I get 300+ of `ERR: Could not resolve ASN for "<ip>"` when running `makeseeds.py`. Wonder if anyone can reproduce this.

Reply:
I also get a absolute tone of "Could not resolve"

Reply:
@knst @ogabrielides could you guys confirm that `nodes_main.txt` list is the same for you too? Or do you get a different one?

In my case it is like that: [nodes_main.txt](https://github.com/dashpay/dash/files/14268508/nodes_main.txt) [diff.txt](https://github.com/dashpay/dash/files/14268526/diff.txt)

I wonder if it's different on every machine or is there at least some determinism...

Reply:
@UdjinM6 I have pretty long diff too, partially matches with you: https://github.com/dashpay/dash/commit/e33a84a08e6f87699b73329f97f03eda2adbc5e4